database/schema.sql

```diff
@@ -44,6 +44,7 @@ CREATE TABLE body_assignment (
     at TIMESTAMP NOT NULL,
     azimuth FLOAT NOT NULL,
     altitude FLOAT NOT NULL,
+    distance_km FLOAT NOT NULL,
```

Comment:
A good addition, distance is an import factor that dictates whether a body is visible to a given subscriber. Not sure why this wasn't done initially or why I didn't catch this. LGTM.

Reply:
Good update